client/src/invoker/http/transaction.rs

```diff
@@ -0,0 +1,47 @@
+use crate::entities::Transaction;
+use crate::invoker::URL;
+use reqwest::{Client, Response};
+use std::path::Path;
+
+/// Структура объекта вызова для работы с транзакциями.
+pub struct TransactionInvoke {
+    client: Client,
+}
+
+impl TransactionInvoke {
+    /// Конструктор.
+    pub fn new(client: Client) -> Self {
+        Self { client }
+    }
+}
+
+/// Интерфейс работы с транзакциями.
+#[trait_variant::make(IntFactory: Send)]
```

Comment:
```suggestion
#[trait_variant::make(Send)]
```

Короче везде аналогично, где я пропустил